rest/route/host_spawn.go

```diff
@@ -1296,9 +1296,15 @@ func validateID(id string) (string, error) {
 func makeSpawnHostSubscription(hostID, subscriberType string, user *user.DBUser) (model.APISubscription, error) {
 	var subscriber model.APISubscriber
 	if subscriberType == event.SlackSubscriberType {
+
+		target := fmt.Sprintf("@%s", user.Settings.SlackUsername)
```

Comment:
only do this if the if check below fails

Reply:
same as above, having it overwrite it if it's available feels more intuitive to me.

Reply:
It saves a blank initialization and else statement